crawl-ref/source/player.cc

```diff
@@ -8817,7 +8817,8 @@ bool temperature_tier (int which)
 
 bool temperature_effect(int which)
 {
-    switch (which)
+    if (you.species == SP_LAVA_ORC)
+    	switch (which)
```

Comment:
the block under switch needs to be indented to the same level as switch like this:

switch (which)
{
    case LORC
...